version.go

```diff
@@ -1,19 +1,27 @@
 package main
 
 import (
+	_ "embed"
+	"strings"
+
 	goversion "github.com/hashicorp/go-version"
 )
 
-// The main version number that is being run at the moment.
-var version = "0.0.0"
+var (
+	// The next version number that will be released. This will be updated after every release
+	// Version must conform to the format expected by github.com/hashicorp/go-version
+	// for tests to work.
+	// A pre-release marker for the version can also be specified (e.g -dev). If this is omitted
+	// then it means that it is a final release. Otherwise, this is a pre-release
+	// such as "dev" (in development), "beta", "rc1", etc.
+	//go:embed version/VERSION
+	fullVersion string
```

Comment:
Nitpick: Given that the version will be hand-crafted, is it worth calling this `rawVersion` or something like that, to indicate that the version was not checked for validity/correctness yet?